exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -390,4 +391,15 @@
 
   String DYNAMIC_UDF_SUPPORT_ENABLED = "exec.udf.enable_dynamic_support";
   BooleanValidator DYNAMIC_UDF_SUPPORT_ENABLED_VALIDATOR = new BooleanValidator(DYNAMIC_UDF_SUPPORT_ENABLED, true, true);
+
+  /**
+   * Option whose value is a long value representing the number of bits required for computing ndv (using HLL)
+   */
+  LongValidator NDV_MEMORY_LIMIT = new PositiveLongValidator("exec.statistics.ndv_memory_limit", 30, 20);
+
+  /**
+   * Option whose value represents the current version of the statistics. Decreasing the value will generate
+   * the older version of statistics
+   */
+  LongValidator STATISTICS_VERSION = new NonNegativeLongValidator("exec.statistics.capability_version", 1, 1);
```

Comment:
Also, the version in the option will change to the latest for any changes to the statistics. However, this option will give users flexibility for scenarios described in the earlier commit.